README.rst

```diff
@@ -3,216 +3,62 @@ Karapace
+as producing and consuming messages and perform administrative cluster work, all the while using the
+language of the WEB.
+
+Karapace is compatible with Schema Registry 6.1.1 on API level.
 
 Features
 ========
 
-* Schema Registry and Rest Proxy aim to be API compatible with the pre-existing proprietary
-  Confluent Schema Registry and Kafka Rest Proxy
 * Drop in replacement both on pre-existing Schema Registry / Kafka Rest Proxy client and
   server-sides
 * Moderate memory consumption
 * Asynchronous architecture based on aiohttp
+* Supports Avro and JSON Schema. Protobuf development is tracked with `Issue 67`_.
 
-
-Overview
-========
-
-Karapace supports the storing of schemas in a central repository, which
-clients can access to serialize and deserialize messages.  The schemas also
-maintain their own version histories and can be checked for compatibility
-between their different respective versions.
-
-Karapace rest provides a RESTful interface to your Kafka cluster, allowing you to perform
-tasks such as producing and consuming messages and perform administrative cluster work,
-all the while using the language of the WEB.
-
-Karapace is compatible with Schema Registry 6.1.1 on API level with the following exceptions
-
-* Protobuf schemas are unsupported
+.. _Issue 67: https://github.com/aiven/karapace/issues/67
 
 Setup
 =====
 
-Karapace is a Python project, and requires Kafka for its backend storage. There is also a `Docker setup for development`_.
-
-Requirements
+Using Docker
 ------------
 
-Karapace requires Python 3.6 or later and some additional components in
-order to operate:
-
-* aiohttp_ for serving schemas over HTTP in an asynchronous fashion
-* avro-python3_ for Avro serialization
-* kafka-python_ to read, write and coordinate Karapace's persistence in Kafka
-* raven-python_ (optional) to report exceptions to sentry
-* aiokafka_ for some components of the rest proxy
-
-.. _`aiohttp`: https://github.com/aio-libs/aiohttp
-.. _`aiokafka`: https://github.com/aio-libs/aiokafka
-.. _`avro-python3`: https://github.com/apache/avro
-.. _`kafka-python`: https://github.com/dpkp/kafka-python
-.. _`raven-python`: https://github.com/getsentry/raven-python
-
-Developing and testing Karapace also requires the following utilities:
-requests_, flake8_, pylint_ and pytest_.
-
-.. _`flake8`: https://flake8.readthedocs.io/
-.. _`requests`: http://www.python-requests.org/en/latest/
-.. _`pylint`: https://www.pylint.org/
-.. _`pytest`: http://pytest.org/
-
-Karapace has been developed and tested on modern Linux x86-64 systems, but
-should work on other platforms that provide the required modules.
-
+To get you up and running with the latest release of Karapace, a docker setup is available::
 
-Building
---------
+    docker-compose -f ./container/docker-compose.yml up -d
 
-To build an installation package for your distribution, go to the root
-directory of a Karapace Git checkout and run::
-
-  python3 setup.py bdist_egg
```

Comment:
`egg`s are being phased out. Just removed this and instructed to run `pip setup.py install` for source installations.